ocpp1.6/securefirmware/signed_update_firmware.go

```diff
@@ -0,0 +1,94 @@
+package securefirmware
+
+import (
+	"reflect"
+
+	"github.com/lorenzodonini/ocpp-go/ocpp1.6/types"
+	"gopkg.in/go-playground/validator.v9"
+)
+
+const SignedUpdateFirmwareFeatureName = "SignedUpdateFirmware"
+
+type SignedUpdateFirmwareFeature struct{}
+
+func (e SignedUpdateFirmwareFeature) GetFeatureName() string {
+	return SignedUpdateFirmwareFeatureName
+}
+
+func (e SignedUpdateFirmwareFeature) GetRequestType() reflect.Type {
+	return reflect.TypeOf(SignedUpdateFirmwareRequest{})
+}
+
+func (e SignedUpdateFirmwareFeature) GetResponseType() reflect.Type {
+	return reflect.TypeOf(SignedUpdateFirmwareResponse{})
+}
+
+// Indicates whether the Charging Station was able to accept the request.
+type UpdateFirmwareStatus string
+
+const (
+	UpdateFirmwareStatusAccepted           UpdateFirmwareStatus = "Accepted"
+	UpdateFirmwareStatusRejected           UpdateFirmwareStatus = "Rejected"
+	UpdateFirmwareStatusAcceptedCanceled   UpdateFirmwareStatus = "AcceptedCanceled"
+	UpdateFirmwareStatusInvalidCertificate UpdateFirmwareStatus = "InvalidCertificate"
+	UpdateFirmwareStatusRevokedCertificate UpdateFirmwareStatus = "RevokedCertificate"
```

Comment:
`InvalidCertificate` and `RevokedCertificate` are not supported on the v1.6 extension

Reply:
I can confirm that these are supported by v1.6

(Page: 62)
![Screenshot 2024-09-16 at 18 51 11](https://github.com/user-attachments/assets/6ae5b996-3711-41ca-8284-ba45eaa05d75)

Reply:
Agreed on second-look 👍